src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -179,7 +185,7 @@ export const IframeRuntime: React.FC<IProps> = forwardRef((props, ref) => {
       }
     };
   }, [url, authoredState, report, initialInteractiveState, setNewHint, getFirebaseJWT, setSupportedFeatures,
-      setSendCustomMessage, showModal, closeModal]);
+      setSendCustomMessage, showModal, closeModal, setNavigation]);
```

Comment:
There will be a trivial conflict here with my outstanding PR #74.

Reply:
Yeah, I was assuming so after reviewing your earlier PR.